ext/js/display/search-display-controller.js

```diff
@@ -313,6 +319,32 @@ export class SearchDisplayController {
         this._updateClipboardMonitorEnabled();
     }
 
+    /**
+     * @param {Event} event
+     */
+    _onProfileSelectChange(event) {
+        const node = /** @type {HTMLInputElement} */ (event.currentTarget);
+        const value = parseInt(node.value, 10);
+        if (typeof value === 'number' && Number.isFinite(value) && value >= 0 && value <= /** @type {import('settings').Options} */ (this._optionsFull).profiles.length) {
```

Comment:
Instead of using `this._optionsFull` here, re-fetch the full settings object again.

Rationale: settings page can be opened twice or changed from other locations, so having an up-to-date copy to check against is beneficial.